pcsx2/PAD/Linux/PAD.h

```diff
@@ -32,12 +32,28 @@
 #include <windowsx.h>
 
 #else
-/*
 #include <X11/Xlib.h>
 #include <X11/Xutil.h>
 #include <X11/keysym.h>
-*/
+// x11 is dumb like that
+#undef DisableScreenSaver
```

Comment:
Why move from keyboard to the general header?

Reply:
The types are used in this file now in order to define the `GSDisplayHandle` struct.

Reply:
Ah right, my bad